### PR TITLE
Comparison interface for participating and non-participating candidates

### LINKED ISSUE #1083: Design comparison interface

### LINKED ISSUE #1087: Implement comparison interface

### PR DESCRIPTION
**Pull request by jaronheard**
Fixes #1083, fixes #1087

- [x] Change to use actual non-participating candidates (using Sarah for testing)
- [x] Determine an actual design for participating / non-participating candidate status to replace 🌹/🥀
- [x] Add number of participants in each category (participating / non-participating)
- [x] Add toggle for participating / non-participating / all (default to participating)
- [x] Add special comparison view for the runoff elections (Mayor's race)
- [x] Clean-up and tests for selectors in publicData.js
- [x] Handle bundled cash contributions $100 and under

**Current screenshot**
full_page_screenshot

### CODE REVIEW

**Comment by jaronheard** [comment]
This demonstrates an initial implementation separating out participating and non-participating candidates. To test, comment out all lines of `export function baseUrl()` in `app/src/api/api.js` except the final return statement so it uses the production API.

**Comment by ghost** [comment]
Maybe a ✅/empty and the column is titled "OAE participant"

**Comment by jaronheard** [comment]
Note -- need to make sure to handle miscellaneous bundled cash contributions $100 or under. These contributions are being are being geocoded to the ORESTAR address (255 Capitol St NE Suite 501, Salem, OR 97310) -- see https://github.com/hackoregon/openelections-lambdas/pull/8 for details.

**Comment by jaronheard** [comment]
Updated design per @Dianna and integrated the new non-participating candidate API. This uses QA fake data for all participating candidates.

image

**Comment by jaronheard** [comment]
Updated design --
# Default screen
localhost_4000_ (2)

# Showing all candidates
localhost_4000_ (3)

**Comment by jaronheard** [comment]
Updated design 

localhost_4000_ (4)

Need to add info and share icons

**Comment by jaronheard** [comment]
Ok, this is getting really close and is ready for review. [Temporary preview site](http://dev.jaronheard.com/) is live using QA data.

# Comparison View
dev jaronheard com__financing=all (1)

# Desktop View
dev jaronheard com__financing=all

| Mobile Comparison View | Mobile View |
| ------------- | ------------- |
| dev jaronheard com_(iPhone 6_7_8) (1) | dev jaronheard com_(iPhone 6_7_8)  |

**Review by andrewbiang888 — APPROVED** [review]

File: app/yarn.lock [hunk 1]
```diff
@@ -11118,18 +9820,21 @@ material-colors@^1.2.1:
   integrity sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg==
 
 material-table@^1.48.0:
-  version "1.48.0"
-  resolved "https://registry.yarnpkg.com/material-table/-/material-table-1.48.0.tgz#3a4baffc915ffd700caf26cb3b272e238cc9cfe1"
-  integrity sha512-OXI8gotPGtFtcSLLXkFzrmPMO8DK5irUd24vq+siTwYPp+kInOM1T5Vrb/pqbR6wQOA2bgWTmwxKGe4GZ1xVcw==
+  version "1.69.0"
```
Comment (jaronheard) [inline app/yarn.lock:9823]: @andrewbiang888 I'm seeing 1.69.0 herer

**Pull request merged**